### ISSUE TITLE
Current ACSE seems like doesnt supports korean language, or broken?

### ISSUE THREAD

**Issue by Tempacc129** [Cuyler36/ACSE#26]
This program always returns IndexOutOfRangeException when i tries to open ACNL:WA savedata. can anyone check this issue?
`
		StackTrace	"   위치: ACSE.Classes.Utilities.PatternUtility.GeneratePalettePreview(UInt32[] Palette, Int32 SelectedColor, UInt32 Width, UInt32 Height) 파일 C:\\Users\\NJ\\Downloads\\ACSE-master\\Classes\\Utilities\\PatternUtility.cs:줄 134\r\n   위치: ACSE.MainForm.Reload_Player(Player Player) 파일 C:\\Users\\NJ\\Downloads\\ACSE-master\\Forms\\MainForm.cs:줄 1440\r\n   위치: ACSE.MainForm.<SetupEditor>d__84.MoveNext() 파일 C:\\Users\\NJ\\Downloads\\ACSE-master\\Forms\\MainForm.cs:줄 728\r\n--- 예외가 throw된 이전 위치의 스택 추적 끝 ---\r\n   위치: System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   위치: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   위치: System.Runtime.CompilerServices.TaskAwaiter.GetResult()\r\n   위치: ACSE.MainForm.<openToolStripMenuItem_Click>d__179.MoveNext() 파일 C:\\Users\\NJ\\Downloads\\ACSE-master\\Forms\\MainForm.cs:줄 3998"	string`

**Comment by Cuyler36** [comment]
Hmm that's a strange error. Are you using the latest release, you did you build the project from the current source?

**Comment by Tempacc129** [comment]
yeah, compiled directly from your latest codes :p

**Comment by Cuyler36** [comment]
I haven't gotten that error with Welcome Amiibo. Would you mind sending me your save file so I can try to reproduce it?

**Comment by Tempacc129** [comment]
sure, https://www.sendspace.com/file/m02b41

**Comment by Cuyler36** [comment]
I pushed an update that should prevent the error from happening to you. Try rebuilding the program from the latest source, and let me know how it goes.

**Comment by Tempacc129** [comment]
huh, may i ask what version of .net framework have u installed?

**Comment by Tempacc129** [comment]
That fixed it tho,but im also curious why it's only happening to me lol

**Comment by Tempacc129** [comment]
Re-Opening this issue due to another bugs i found :/

https://imgur.com/a/fdY0bi9

**Comment by Cuyler36** [comment]
Something is different with your .NET/ Computer setup. I have .NET 4.5 installed. None of those things are broken for me.

**Comment by Tempacc129** [comment]
Do you have any clue that what sort of .net setup makes up this issue? i really have no idea about this :/

**Comment by Cuyler36** [comment]
No, but I'd try to uninstall/re-install your .NET packages. It could be your locale/Windows version. I haven't tested it on anything other than Windows 10. I know it should work on Windows 7 and 8 though.
